example/ipsec_offload/run_left.sh

```diff
@@ -2,13 +2,15 @@
 #
 
 ./odp_ipsec_offload -f 64 -i dpni.1,dpni.2 \
```

Comment:
As discussed, interface names should be generic (loop or virtual interfaces). Let's check first if the application is duplicate of other ipsec test applications.

Reply:
The ipsec_offload application code seems to be mostly duplicate from ipsec_api and the app hasn't event started properly for the past two years (odp_schedule_config() is called too late). So, I thing it's safe to remove the whole application.

Reply:
I looked at the ipsec_offload example and agree that we should just remove it. So let's not bother to change its parameter format as part of this PR.

Reply:
Ack. Dropping the changes to ipsec_offload application from this PR.